app/server/lib/ActiveDoc.ts

```diff
@@ -1615,7 +1615,8 @@ export class ActiveDoc extends EventEmitter {
 
   public async getAccessToken(docSession: OptDocSession, options: AccessTokenOptions): Promise<AccessTokenResult> {
     const tokens = this._server.getAccessTokens();
-    const userId = getUserId(docSession);
+    const user = await this._granularAccess.getUser(docSession);
```

Comment:
Can you walk me through what this fixes?

Reply:
It applied ACL related accessses instead of connected user's.